extensions/2.0/Vendor/EXT_feature_metadata/README.md

```diff
@@ -212,11 +212,11 @@ Feature IDs may also be assigned to individual instances when using the [`EXT_me
 }
 ```
 
-## Feature Metadata
+## Feature Properties
```

Comment:
just to make sure this is intentional, we're saying the concept is called "feature properties" (I guess to distinguish it from the corresponding class properties?)

Reply:
Partly I'm trying to limit the use of the term "metadata" since I don't have a crisp definition for it in this context. But "Feature Properties" and "Class Properties" do sound like reasonable choices to disambiguate these where necessary. Also hoping to have parallelism between these two top-level headers: 

- Feature Identification
- Feature Properties

Or, might be more better to say "Feature IDs" and "Feature Properties"? Presumably our readers don't know all these terms beforehand, but the more we can do to define terms clearly and reuse them consistently, the better.

Reply:
Sounds good, just wanted to make sure I understand the intent here.